src/07-even-more-loops.js

```diff
@@ -96,7 +142,15 @@ function evenArray(array) {}
  *  findSmallest([ 0, 11, -2, 5 ]);
  *  //> -2
  */
-function findSmallest(array) {}
+function findSmallest(array) {
+  let newArr = [];
+  for(let i =0; i <= array.length-1; i++) {
+    newArr.push(array[i]);
+  }
+return Math.min(...newArr)
```

Comment:
Make very sure we know what ` ... ` is, what its called, and how it works.  This is a slick tool to add to your toolbox but make sure you know what it is.